nsv13/code/modules/munitions/ammunition/magazines.dm

```diff
@@ -55,12 +55,20 @@
 	caliber = "mm30.12"
 	max_ammo = 300
 
+/obj/item/ammo_box/magazine/nsv/pdc/rifle
+	max_ammo = 1
+
```

Comment:
```suggestion
/obj/item/ammo_box/magazine/internal/boltaction/pdc
	max_ammo = 3
```
You can probably give them more bullets in the internal magazine, and it should also be a subtype of /magazine/internal as it's inside the gun. (this also lets them load in multiple bullets at once, which _can_ be changed)